profiles/common/modules/custom/nexteuropa_dgt_connector/tmgmt_dgt_connector/README.md

```diff
@@ -75,7 +75,7 @@ In the translation checkout page, the translator ```TMGMT DGT Translator``` must
 
 # Configuration
 
-* Install and configure [NextEuropa DGT Connector](https://github.com/ec-europa/platform-dev/tree/master/profiles/common/modules/features/nexteuropa_dgt_connector);
+* Install and configure [NextEuropa DGT Connector](https://github.com/ec-europa/platform-dev/tree/master/profiles/common/modules/custom/nexteuropa_dgt_connector);
```

Comment:
master is no longer available, i suggest changing it to release-2.6/profiles/...